scripts/Microsoft.Xaml.Behaviors.Uwp.Managed.nuspec

```diff
@@ -15,11 +15,11 @@
         <tags>Behavior Action Behaviors Actions Blend Managed C# Interaction Interactivity Interactions</tags>
         <repository type="git" url="http://go.microsoft.com/fwlink/?LinkID=651678" />
         <dependencies>
-            <group targetFramework="net8.0-windows10.0.19041" />
```

Comment:
Leave all of these to 19041 in this file.

Reply:
The problem is that the build runs on a [Windows 2025 image](https://github.com/actions/runner-images/blob/win25/20251014.59/images/windows/Windows2025-Readme.md), which doesn't have the 19041 SDK. I'd need to somehow switch to the [Windows 2022 runner](https://github.com/actions/runner-images/blob/win25/20251014.59/images/windows/Windows2022-Readme.md) image.